content/en/docs/ops/setup/injection-concepts/index.md

```diff
@@ -31,15 +31,11 @@ the three configuration items. The security rules above cannot be overridden.
 
 | `namespaceSelector` match | default `policy` | Pod override annotation `sidecar.istio.io/inject` | Sidecar injected? |
 |---------------------------|------------------|---------------------------------------------------|-----------|
-| yes                       | enabled          | true                                              | yes       |
+| yes                       | enabled          | true (default)                                    | yes       |
 | yes                       | enabled          | false                                             | no        |
-| yes                       | enabled          |                                                   | yes       |
 | yes                       | disabled         | true                                              | yes       |
-| yes                       | disabled         | false                                             | no        |
-| yes                       | disabled         |                                                   | no        |
+| yes                       | disabled         | false (default)                                   | no        |
 | no                        | enabled          | true                                              | no        |
```

Comment:
Even though it doesn't change anything, I would specify (default) for the no match case too
```suggestion
 | no                        | enabled          | true  (default)                                   | no        |
```